azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py

```diff
@@ -73,7 +73,11 @@ def test_create_bundle(init_setup, bundle_dir, mq_traces, ops_service, tracked_f
             walk_result[path.join(BASE_ZIP_PATH, namespace, "mq")]["folders"] = []
 
     # Level 2 and 3 - bottom
-    assert len(walk_result) == (len(expected_services) + int("clusterconfig" in expected_services))
+    actual_walk_result = (len(expected_services) + int("clusterconfig" in expected_services))
+    if lnm_namespace:
```

Comment:
need to rewrite this since the lmn namespace is pulled from the support bundle files (and this is checking support bundle files)

This should somehow get info from the service instead

Reply:
added in actual lnm check